superset/assets/spec/javascripts/dashboard/fixtures.jsx

```diff
@@ -169,3 +169,141 @@ export const {
   datasources: {},
   user_id: '1',
 });
+
+export const mockSliceEntities = {
```

Comment:
@graceguo-supercat can we actually delete this file? all of the fixtures for the new dashboard are within `spec/javascripts/dashboard/fixtures/` and are separated by type (not all in one file). there's actually one for `mockSliceEntities` but you could add to it.